src/mono/mono/metadata/bundled-resources-internals.h

```diff
@@ -0,0 +1,120 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#ifndef __MONO_METADATA_BUNDLED_RESOURCES_INTERNALS_H__
+#define __MONO_METADATA_BUNDLED_RESOURCES_INTERNALS_H__
+
+#include <assert.h>
```

Comment:
Since this is an internal runtime header, we could just include use g_assert and drop the include of assert.h

Reply:
Right, since we are no longer having a 1-to-1 mirror with the "header" in the emit bundle task, removed the assert altogether.